### DISCUSSION TITLE
High resolution monitor support

### DISCUSSION THREAD

**Discussion by lsces** [pdfarranger/pdfarranger#1129] [Q&A]
Was going to add this to my other post, but it warrants it's own - I'm using a high res monitor which helps the display of 30+ page pdf's somewhat, but the top bar icons are very small. A problem which is not restricted to PDF Arranger and I had originally reported this to the eclipse project. On PDF Arranger there are only a few icons and I know what they are so I can hit the spot, but it would be nice if the spot was not 16x16 on a 3840x2160 screen ... 
![pdfarranger_top_bar](https://github.com/user-attachments/assets/40b766b5-9833-4dae-8ad9-7f18582e601e)
I think I have established that this is a platform related problem rather than the applications, and it needs to be fixed really at the Qt level? The windows icons top right are the same small size as the tools with the same problem hitting them.

**Comment by dreua** [comment]
Which distro / desktop / window manager are you using? PDF Arranger uses Gtk, not Qt. Afaik Eclipse uses its own Java based framework. This already sounds much like "not our bug". I don't own a high res display so theres not much more I could say or do about this unfortunately.

We already improved preview generation for high res in #606 and I haven't heard of any other issues in the area since then.
